selfdrive/car/honda/fingerprints.py

```diff
@@ -395,6 +395,46 @@
       b'77959-TMM-F040\x00\x00',
     ],
   },
+  CAR.HONDA_ODYSSEY_BOSCH: {
+    (Ecu.programmedFuelInjection, 0x18da10f1, None): [
+      b'37805-5MR-C730\x00\x00',  # 2021 Touring
+      b'37805-5MR-C910\x00\x00',  # 2022 Touring
+      b'37805-5MR-C920\x00\x00',  # 2022 Elite
+      b'37805-5MR-C110\x00\x00',  # 2023 Sport
+    ],
```

Comment:
Remove this block: `programmedFuelInjection` is in [extra_ecus](https://github.com/commaai/openpilot/blob/7de2aac3c93caa20213894e574f06bd45f789335/selfdrive/car/honda/values.py#L308-L316) for Honda, not for fingerprinting.

This and `combinationMeter` are the cause of your build_release CI failure.

```suggestion
```